lib/src/util/map_util.dart

```diff
@@ -68,6 +68,58 @@ Map getPropsToForward(Map props, {
   return propsToForward;
 }
 
+/// Adds unconsumed props to a passed in [Map] reference ([propsToUpdate]).
+///
+/// Based upon configuration, the function will overlook [props] that are not
+/// meant to be passed on, such as non-DOM props or specified values.
+void forwardUnconsumedProps(Map props, {
```

Comment:
Should this be advertised as a replacement for `getPropsToForward`?  If so, should `getPropsToForward` be deprecated?

Reply:
That's a good point @aaronlademann-wf. I tried to look around and see if anyone was using `getPropsToForward` outside of the context of `addProps`, and there were a [few instances](https://github.com/search?p=1&q=getPropsToForward&type=Code).

I feel like the functions are different enough that it could be beneficial to have a more pure version of finding the unconsumed props without mutating a reference directly. So I'd lean towards it not being a replacement, but is that logic sound or am I missing anything?

Reply:
No, keeping the other one around sounds reasonable.